tests/integration/redis-cli.tcl

```diff
@@ -109,7 +109,7 @@ start_server {tags {"cli"}} {
     test_interactive_cli "INFO response should be printed raw" {
```

Comment:
As with redis-benchmark, this filename is shown to users during tests and should be renamed to reflect our filenames.